replicatedhq/ship
# Regularlize yaml lib usage

What I Did
------------
We used both `ghodss/yaml` and `gopkg.in/yaml.v2`. Now we use `ghodss/yaml` in one place and `gopkg.in/yaml.v3` everywhere else.

How I Did it
------------
much finding and replacing.

How to verify it
------------
Things still work. `Gopkg.lock` no longer includes `gopkg.in/yaml.v2` as a top-level import.

Description for the Changelog
------------



Picture of a Ship (not required but encouraged)
------------




![USS Hancock (CV-19)](https://upload.wikimedia.org/wikipedia/commons/7/7f/USS_Hancock_%28CVA-19%29_off_Pearl_Harbor_1968.jpg "USS Hancock (CV-19)")
